### ISSUE TITLE
[Bug][a11y] Consider increasing text contrast ratio in 'Share tab'

### ISSUE THREAD

**Issue by neha-b2001** [mozilla-mobile/fenix#18768]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
1. Install and open Google Accessibility Scanner
2. In the homescreen, click on the square **tab counter icon** in the bottom right part of homescreen
3. Click any one of the tabs displayed in the list for slightly long
4. Click on the **share** icon
### Expected behavior
No suggestions are made.
### Actual behavior
Google Accessibility Scanner makes suggestions to **increase text contrast ratio at multiple places in the page**. 

**All suggestions**
![image](https://user-images.githubusercontent.com/67039214/113439740-c4f56d80-9408-11eb-9960-6e38c85817dd.png)

**For URL**
![image](https://user-images.githubusercontent.com/67039214/113440093-5fee4780-9409-11eb-8161-498f3b75f5b6.png)

**For RECENTLY USED**
![image](https://user-images.githubusercontent.com/67039214/113440327-d8550880-9409-11eb-919d-cd19909477ff.png)

**For App names**
![image](https://user-images.githubusercontent.com/67039214/113440613-65985d00-940a-11eb-9475-d4e56e806b94.png)

### Device information

* Device vendor / model and Android version: Redmi Note 8 Pro (Android 10)
* Firefox for Android version: Nightly 210331 17:00 (Build #2015801931)

**Comment by neha-b2001** [comment]
I am an Outreachy applicant, hence tagging @eliserichards. Please take a look if possible.

**Comment by ebalazs-sv** [comment]
I can reproduce this issue on Nightly 4/6, Beta 88.0.0-beta.2 and RC 87.0.0-rc.1 with Pixel 2 (Android 9).

**Comment by topotropic** [comment]
Thanks!

- For the tiny URL we can use `secondary_text_dark_theme (#AFAFBB)`
- For "Recently used" and app names `secondary_text_light_and_dark_theme (#52525E)`

**Comment by neha-b2001** [comment]
@topotropic Okay 👍 
I'll do the changes and make a PR.

**Comment by lobontiumira** [comment]
Verified on the 6/7 Nightly build with Samsung Galaxy Note 8 (Android 9), and:
- using the light theme gives no suggestions when scanning with Accessibility scanner,
- using the dark theme gives multiple suggestions:
"Text contrast
org.mozilla.fenix:id/appName
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text."

![screenshot_Firefox Nightly_2021-06-07-17 03 42](https://user-images.githubusercontent.com/40694158/121030800-7bcbec00-c7b2-11eb-9291-5ddc330da8cb.png)

Also, in dark theme, the "RECENTLY USED" text, and the recently used apps's names aren't visible at all.

**Comment by marcel263** [comment]
still reproducible in latest nightly

**Comment by mcarare** [comment]
@lobontiumira Can this be closed, are there any issues left to fix? Thank you!

**Comment by lobontiumira** [comment]
This issue still reproduces on the Firefox Nightly from 12/10 using a Samsung Galaxy Note 8 (Android 9):
* with **light theme**, the A11y Scanner suggests one modification:

"Image contrast
org.mozilla.fenix:id/closeButton
The image's contrast ratio is 1.44. This ratio is based on an estimated foreground color of #20123A and an estimated background color of #363541. Consider increasing this ratio to 3.00 or greater."

* with **dark theme**, the scanner suggests multiple improvements:

"Text contrast
org.mozilla.fenix:id/apps_link_header
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

Text contrast
org.mozilla.fenix:id/appName
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

Text contrast
org.mozilla.fenix:id/appName
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

Text contrast
org.mozilla.fenix:id/appName
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text. This item may be only partially visible within a scrollable container.

Text contrast
org.mozilla.fenix:id/appName
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

Text contrast
org.mozilla.fenix:id/appName
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

Text contrast
org.mozilla.fenix:id/appName
The item's text contrast ratio is 1.66. This ratio is based on an estimated foreground color of #52525E and an estimated background color of #32313C. Consider using colors that result in a contrast ratio greater than 4.50 for small text, or 3.00 for large text. This item may be only partially visible within a scrollable container."

![a11y scanner](https://user-images.githubusercontent.com/40694158/145542594-88fa3d1b-7df2-4f0b-9c19-ea85532569a3.png)
